evm-adapters/testdata/CheatCodes.sol

```diff
@@ -358,7 +360,32 @@ contract CheatCodes is DSTest {
     // after expectRevert
     function testFailExpectRevert3() public {
         hevm.expectRevert("revert");
-    }  
+    }
+
+    function testMockCall() public {
+        hevm.mockCall(address(0xbeef), abi.encode("wowee"), abi.encode("epic"));
+        (bool ok, bytes memory ret) = address(0xbeef).call(abi.encode("wowee"));
```

Comment:
Would this also work if you called another contract, which calls 0xbeef with the calldata? Could be worth adding a test for it, if so.

Reply:
it should but prolly good to add test yes

Reply:
test added